pkg/storer/migration/step_06.go

```diff
@@ -160,5 +161,21 @@ func addStampHash(logger log.Logger, st transaction.Storage) (int64, int64, erro
 		return 0, 0, err
 	}
 
+	batchRadiusCnt, err := st.IndexStore().Count(&reserve.BatchRadiusItem{})
+	if err != nil {
+		return 0, 0, err
+	}
+
+	chunkBinCnt, err := st.IndexStore().Count(&reserve.ChunkBinItem{})
+	if err != nil {
+		return 0, 0, err
+	}
+
+	if batchRadiusCnt != chunkBinCnt {
+		return 0, 0, fmt.Errorf("index counts do not match, %d vs %d", batchRadiusCnt, chunkBinCnt)
+	}
```

Comment:
This should be before the migration starts.
Then at the end, we compare that `seenCnt == doneCnt == batchRadiusCnt == chunkBinCnt`